app/components/single-set/score.hbs

```diff
@@ -0,0 +1,23 @@
+<div ...attributes data-test-score class="single-set-score {{this.statusClass}}">
+    {{#if @isEnabled}}
+        <button 
+            data-test-minus-icon 
+            class="icon minus-icon" 
+            type="button" 
+            {{on 'click' (fn this.decrement @set @scoreParam @callback)}}
```

Comment:
don't need to pass @set, @scoreParam, @callback to this action. We can get those values from this.args